ci/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM alpine:edge
+
+RUN apk add --no-cache g++ binutils-gold clang \
```

Comment:
I don't think you need `gold` though, LDC doesn't use it (doesn't quite work on Alpine).
See https://gitlab.alpinelinux.org/alpine/aports/-/blob/master/community/ldc/01-conf.patch